includes/class-wc-payments-upe-checkout.php

```diff
@@ -137,11 +133,7 @@ public function register_scripts() {
 			$script_dependencies[] = 'woocommerce-tokenization-form';
 		}
 
-		if ( WC_Payments_Features::is_upe_deferred_intent_enabled() ) {
-			$script = 'dist/upe_with_deferred_intent_creation_checkout';
-		} else {
-			$script = 'dist/upe_checkout';
```

Comment:
Note for our future selves: `dist/upe_checkout` seems to no longer be used. It looks like its reference in `webpack/shared.js` can be removed, as well as `client/checkout/classic/upe.js` and every other frontend dependency imported in `client/checkout/classic/upe.js` that is not shared

Reply:
Mentioned in the last bullet point in #7780